akka-sample-sharding-scala/README.md

```diff
@@ -1,39 +1,153 @@
-This tutorial contains a sample illustrating [Akka Cluster Sharding](http://doc.akka.io/docs/akka/current/scala/cluster-sharding.html#an-example).
+# Cluster Sharding sample
 
-## Example overview
+The KillrWeather sample illustrates how to use [Akka Cluster Sharding](http://doc.akka.io/docs/akka/current/scala/cluster-sharding.html).
+It also shows the basic usage of
+[Akka HTTP](https://doc.akka.io/docs/akka-http/current/index.html), and more coming soon, such as
```

Comment:
I'm not sure this sample should be expanded to cover more things. Risk that it becomes too complicated and difficult to understand the primary topic, which is Sharding.

Reply:
I was thinking the same after pushing this. I'll remove.